content/07-native-tokens/06-exchange-multi-asset-management-scenarios.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Exchange multi-asset management scenarios
+metaTitle: Exchange multi-asset management scenarios
+---
+
+Native assets follow a set of predefined rules known as *minting policies*, which are defined by users who want to create a new asset. These policies set the upper and lower limits of what a token can do, who can do it, and might also define the point in the transaction lifecycle at which it can be done.
+
+A minting policy is bound by the network protocol and ledger-defined rules. One such rule is [the minimum UTXO value](https://cardano-ledger.readthedocs.io/en/latest/explanations/min-utxo.html) (the minimum amount of ada that must be sent in a single transaction.) The value is currently set to 1 ada, so to transact any native asset on the Cardano network, a minimum of 1 ada must be included and accounted for within the transaction. This does not mean that you have to send only 1 ada to transact native assets. You can use any amount of ada, so it is important that exchanges and wallets are aware of this, and know what to do in such cases.  
+
+Note: It is possible to verify and pinpoint these specific native asset transactions using [cardano-graphql](https://github.com/input-output-hk/cardano-graphql#overview) or [cardano-rosetta](https://docs.cardano.org/rosetta/learn-rosetta).
+
+## Multi-asset Management Scenarios
+Exchanges listing ada usually might encounter two specific scenarios:
+
+- Scenario 1 - Spending a UTXO with a multi-asset attached
+- Scenario 2 - Unexpectedly received multi-asset in cardano-wallet
+
+### Scenario 1: Spending a UTXO with a multi-asset attached
```

Comment:
I'd suggest using H2 instead of H3, as we later have Scenario2 formatted as H2 - for better formatting clarity